portfolio/src/main/webapp/script.js

```diff
@@ -37,14 +37,13 @@ function addRandomGreeting() {
  //Precondition: Assume directory has images of form dirNameXX.jpg where XX is an integer 
 function getRandomFile(element, path, fileName, numElems){
   
+  //It would be ideal if files didn't have to follow a certain naming convention,
+  //but not sure this is possible with strictly server side JavaScript. After
+  //conferring with help line seems this is indeed impossible. 
+  //TODO: Implement on server side to clean up need for numElems, etc in the directory 
+  //For now copy directly the example code with minor modifications.
```

Comment:
There should be a space between // and the start of the comment.

Reply:
Ok

Reply:
Is this true for all comments (HTML, CSS, etc)?
Like an HTML comment should be <!-- comment --> ?

Reply:
Looking through I realize I haven't been super consistent on this so it would be good to know which is preferred. Looking at the CSS and HTML style guide I don't see a preference.

Reply:
I think all comments should follow the same style.

Reply:
Consistently across all languages I've seen, we prefer to have a space between the comment character and the text. It is odd that the guides don't specify it, but their examples use that form too.